ui/src/app/pages/data/cohort/modifier-page.spec.tsx

```diff
@@ -64,31 +65,30 @@ describe('ModifierPage', () => {
     );
   };
 
-  it('should render', () => {
-    const wrapper = component();
-    expect(wrapper.exists()).toBeTruthy();
+  it('should render', async () => {
+    component();
+    await screen.findByRole('heading', {
+      name: /apply optional modifiers/i,
+    });
   });
 
   it('should display Only Age Event and CATI modifiers for SURVEY', async () => {
     currentCohortSearchContextStore.next({
       domain: Domain.SURVEY,
       item: { modifiers: [] },
     });
-    const wrapper = component();
-    await waitOneTickAndUpdate(wrapper);
-    expect(wrapper.exists()).toBeTruthy();
+    component();
+
+    await waitFor(() => {
+      expect(screen.queryByLabelText('Please Wait')).not.toBeInTheDocument();
+    });
     expect(
-      wrapper.find('[data-test-id="' + ModifierType.AGE_AT_EVENT + '"]').length
+      screen.getAllByTestId(ModifierType.AGE_AT_EVENT).length
     ).toBeGreaterThan(0);
     expect(
-      wrapper.find('[data-test-id="' + ModifierType.NUM_OF_OCCURRENCES + '"]')
-        .length
-    ).toBe(0);
-    expect(
-      wrapper.find('[data-test-id="' + ModifierType.ENCOUNTERS + '"]').length
-    ).toBe(0);
-    expect(
-      wrapper.find('[data-test-id="' + ModifierType.EVENT_DATE + '"]').length
+      screen.queryAllByTestId(ModifierType.NUM_OF_OCCURRENCES).length
```

Comment:
I'd prefer `queryBy` and `not.toBeInTheDocument()` instead

Reply:
I like that too. Thanks for pointing it out.